flent/runners.py

```diff
@@ -1682,7 +1683,13 @@ def check(self):
             if self.ip_version is not None:
                 args.append("-{}".format(self.ip_version))
 
-            args.append(self.host)
+            try:
+                ipaddress.IPv6Address(self.host)
+                host = "[{}]".format(self.host)
+            except ValueError:
+                host = self.host
```

Comment:
Using the ipaddress module for parsing is fine, but let's move this into the normalise_host() utility function. Simply adding a second argument 'bracket_v6' which defaults to 'False' to that should do the trick :)

Reply:
Ah, I didn't notice that, that's a much better solution. :)

Reply:
It just occurred to me that when adding optional parameters, it might be good practice to specify them by name in calls to those functions, so I updated that. I don't know what normal python practice is. I guess as long as the parameter positions don't change it's not critical, but this might also make it more apparent what you're doing in the call itself.